pkg/ruler/ruleset_test.go

```diff
@@ -173,3 +174,51 @@ data:
 		t.Errorf("Got error %v ", report.Message)
 	}
 }
+
+func TestRuleset_Get_intoto(t *testing.T) {
+	var data = `
+---
+apiVersion: apps/v1
+kind: Deployment
+spec:
+  template:
```

Comment:
Deployment is missing labels and match selector so this test may need some tweaking